nautobot/core/ui/object_detail.py

```diff
@@ -732,6 +752,26 @@ def render_label(self, context):
             return bettertitle(context["object"]._meta.verbose_name)
         return super().render_label(context)
 
+    def render_value(self, key, value, context):
+        try:
+            field_instance = context["obj"]._meta.get_field(key)
+        except FieldDoesNotExist:
+            field_instance = None
+
+        if key == "_hierarchy":
+            return render_tree_queryset(value)
+
+        if isinstance(field_instance, URLField):
+            return hyperlinked_field(value)
+
+        if isinstance(field_instance, BooleanField):
+            return render_boolean(value)
+
+        if isinstance(field_instance, ManyToManyField) and field_instance.related_model == ContentType:
+            return render_content_types(value)
```

Comment:
Do we need an `else` here for the case of M2M that *aren't* ContentTypes?

Reply:
No - M2M fields *in general* shouldn't be rendered in an ObjectFieldsPanel because there may be arbitrarily many related objects. Content-type is at least bounded to a maximum of low three-digits. M2M should be rendered as an ObjectsTablePanel instead so that we get pagination and so forth.

Reply:
I think I should open an issue for auto M2M fields as `ObjectFieldsPanel`. Alternatively, perhaps we should display M2M fields only in cases where the `ObjectFieldsPanel` is supplied; in the event that it is not, the M2M field is ignored.